raiden-dapp/src/components/AmountInput.vue

```diff
@@ -1,8 +1,6 @@
 <template>
   <fieldset class="amount-input">
-    <div class="amount-input__label">
-      {{ label }}
-    </div>
+    <div class="amount-input__label">{{ label }}</div>
```

Comment:
Out of curiosity Is this done by hand or is it something done automatically by VSCode?

Reply:
I always use `npm run lint` before committing. It's either VSCode or linter settings.